pkgs/by-name/cr/crewai/package.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "crewai";
+  version = "0.175.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "crewAIInc";
+    repo = "crewAI";
+    tag = version;
+    hash = "sha256-F1hw8MTQDA6eW9Rw+MdV3hJHiTuj1SHwu2c39KirPnI=";
+  };
+
+  build-system = with python3Packages; [
+    hatchling
+  ];
+
+  pythonRelaxDeps = [
+    "chromadb"
+    "json-repair"
+    "litellm"
+    "portalocker"
+    "pyvis"
+  ];
+
+  nativeBuildInputs = with python3Packages; [
+    pythonRelaxDepsHook
+  ];
+
+  dependencies = with python3Packages; [
+    pydantic
+    openai
+    litellm
+    instructor
+    pdfplumber
+    regex
+    opentelemetry-api
+    opentelemetry-sdk
+    opentelemetry-exporter-otlp-proto-http
+    chromadb
+    tokenizers
+    onnxruntime
+    openpyxl
+    pyvis
+    python-dotenv
+    pyjwt
+    click
+    appdirs
+    jsonref
+    json-repair
+    tomli-w
+    tomli
+    blinker
+    json5
+    portalocker
+    uv
+  ];
+
+  dontUsePythonImportsCheck = true;
+  pythonImportsCheck = [ "crewai" ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
```

```suggestion

  nativeCheckInputs = [
    pytestCheckHook
  ];

```
Please, try to run the test suite (if any).